README.md

```diff
@@ -140,46 +159,37 @@ Use the following command to format and lint the code:
 #### Format the code
 
 ```bash
-npm run format
+yarn run format
 
 #OR
 
-yarn run format
+npm run format
 ```
 
 #### Lint the code
 
 ```bash
 # to check the linting issue
 
-npm run lint
-
-#OR
-
 yarn run lint
+#OR
+npm run lint
 
 
 # to fix the linting issue
-
-npm run lint:fix
-
-#OR
-
 yarn run lint:fix
+#OR
+npm run lint:fix
 ```
 
 ### 🧱 Build the App for Production
 
 Use the following command to build the app for production:
 
-```bash
-npm run build
-```
-
-Or
-
 ```bash
```

Comment:
I would keep them separate as it was before. It is easy for some one to copy only the line they want with a single click. Please do this change for all the places where you combined. Otherwise the flow is good.

Reply:
done